assets/css/style.css

```diff
@@ -369,6 +373,11 @@ nav #site-brand a:hover {
     padding: 0;
 }
 
+.navbar .navbar-toggler{
+    position:absolute;
+    right: 10px;
```

Comment:
Hey, I tried using my IDE extension to format the CSS file but ended up making many changes. My extension is formatting a bit differently so this happened.
I am using the prettier code formatter and I also observed that this whitespace inconsistency problem is existing in many places including the ones which are not related to this PR. 
Changing all of them manually will be a big task. What do you suggest?

Reply:
@Neha9849 I see. Yeah, this will cause a lot of changes and make an existing Pull Request difficult to review. Let's postpone the automated code formatter for now, and revisit when you are working on the Open Source Inventory site for #126. For now, it will be good to manually keep the whitespace while working, until we can revisit the code formatting.